### DISCUSSION TITLE
Time series regression

### DISCUSSION THREAD

**Discussion by Ahmed1950S** [timeseriesAI/tsai#301] [Q&A] [answered]
Hello, I'm new to programming and Fastai, because the nature of my project I had to jump into time series.
Can you help me with the following : I'm working with market Data each raw represents one minute, the "0" column is one minute log-returns and the "1" column is for 5 minutes log-returns. I want to make predictions based on 30 minutes intervals. 
I followed all the steps but my learner gives me 0 accuracy. Is my window size too large ?

![Step1](https://user-images.githubusercontent.com/63210263/144881135-456b7db0-2498-4808-9e1f-cc801891d952.jpg)
![Step2](https://user-images.githubusercontent.com/63210263/144881139-096c801d-b530-4c98-8780-96287362f7d1.jpg)
![Step3](https://user-images.githubusercontent.com/63210263/144881174-4dfa460e-b325-4fb5-ab61-0e969002b660.jpg)
![Step4](https://user-images.githubusercontent.com/63210263/144881177-0fd9d8c2-0d2a-40c2-a427-356fcbccf585.jpg)

.

**Comment by Ahmed1950S** [comment]
Is my data fitted to make 30 minute predictions ?

**Comment by oguiza** [comment, marked as answer]
You have set your data to use 30 time steps of history to predict the next one. This will be done for every minute. If you want to make predictions every 30 mins, you should use a stride of 30 (default is 1).

**Reply by Ahmed1950S** [reply]
managed to get it to work, thanks a lot for the help.
